src/Lighthouse/LighthouseHostFactory.cs

```diff
@@ -24,13 +24,11 @@ public static class LighthouseHostFactory
         {
             systemName = systemName ?? Environment.GetEnvironmentVariable("ACTORSYSTEM")?.Trim();
 
-
-            // Set environment variables for use inside Akka.Bootstrap.Docker
-            // If overrides were provided to this method.
-            //if (!string.IsNullOrEmpty(ipAddress)) Environment.SetEnvironmentVariable("CLUSTER_IP", ipAddress);
-
-            //if (specifiedPort != null)
-            //    Environment.SetEnvironmentVariable("CLUSTER_PORT", specifiedPort.Value.ToString());

```

Comment:
We probably need to add this back?

Reply:
I already added it back, but instead of injecting it into the environment variable, I did a config fallback instead